lib/sync-settings.coffee

```diff
@@ -31,6 +31,9 @@ module.exports =
     atom.commands.add 'atom-workspace', "sync-settings:backup", => @backup()
     atom.commands.add 'atom-workspace', "sync-settings:restore", => @restore()
 
+    # Clean up after #36, see #68 for more info.
+    fs.unlink(require('path').join(atom.getConfigDirPath(), 'snippets.coffee'), ->)
```

Comment:
The contents of both files is more then likely different when using this package on multiple computers. If on a remote computer the snippets are updated, the contents will be stored on the `snippets.coffee` file on the local computer, never updating the `snippets.cson`.